File: Array/Largest Element in Array/largestElementInArray.cpp
#include<iostream>
#include<cstdio>
#define maxSize 100
using namespace std;

int main()
{
    int array[maxSize], size;
    int max;

    // Scanning the size of the array
    cout << "Enter the size of the array: ";
    cin >> size;

    // Scanning the elements of the array
    cout << "Enter the " << size << " elements of the array: " << endl;
    for(int i=0; i<size; i++)
    {
        printf("ELement [%d]: ",i);
        cin >> array[i];
    }

    // Printing the elements of the array
    cout << "The input array: " << endl;
    for(int i=0; i<size; i++)
    {
        cout << array[i] << " ";
    }

    // Assigning the first element of the array to max variable
    max = array[0];

    // Checking for elements maximum than value of max variable
    for(int i=0; i<size; i++)
    {
        if(array[i] > max)
        {
            max = array[i];
        }
    }

    // Printing out the result
    cout << "\nThe largest element of input array: " << max << endl;

    return 0;
}

File: Array/Largest Element in Array/largestElementInArray.java
import java.util.*;

public class largestElementInArray 
{
    public static void main(String[] args)
    {
        try(Scanner sc = new Scanner(System.in))
        {
            // Scanning the size of array
            System.out.print("Enter the size of the array: ");
            int size = sc.nextInt();

            int array[] = new int[size];
            // Scanning the elements of array
            System.out.println("Enter the elements of array: ");
            for(int i=0; i<size; i++)
            {
                System.out.print("Element ["+ i +"]: ");
                array[i] = sc.nextInt();
            }

            //Printing the array
            System.out.println("Array: ");
            for(int i=0; i<size; i++)
            {
                System.out.print(array[i] + " ");
            }
            System.out.println();

            // Assigning the first element of array to max variable
            int max = array[0];

            // Computing largest element in array
            for(int i=0; i<size; i++)
            {
                if(array[i] > max)
                {
                    max = array[i];
                }
            }

            // Printing the result
            System.out.println("The largest element of the array: "+max);
        }
    }
}
